packages/flutter_test/lib/src/window.dart

```diff
@@ -660,6 +672,16 @@ class TestFlutterView implements FlutterView {
   /// can only be set in a test environment to emulate different view
   /// configurations. A standard [FlutterView] is not mutable from the framework.
   ///
+  /// This property and [physicalGeometry] are dependent on one another. If both
+  /// properties are set through their test setters, the final result will be
+  /// that [physicalGeometry] determines the location and [physicalSize]
+  /// determines the size of the [physicalGeometry] [Rect]. If only
+  /// [physicalSize] is set, the final result is that the default value of
+  /// [physicalGeometry] determines the location and [physicalSize] determines
+  /// the size of the [physicalGeometry] [Rect]. If only [physicalGeometry]
+  /// is set, it will determine both the location and size of the
+  /// [physicalGeometry] [Rect].
```

Comment:
Could use a macro for everything but the first line since that's duplicated from the other doc?

Reply:
Personally I'm not a fan of macros, they make it hard to read when you're actually in the source code IMO. If you want it, I'll put it in, but I tend to avoid them in general.

Reply:
Your call. We tend to use them to avoid that when one doc location gets updated, the other one goes out of date. You're right, it does degrade reading the source a little bit. We've been pushing for better macro support in IDEs, though; that would at least give you a better experience there.

Reply:
For this case I think I'll leave it as is then. I'll keep macros in mind for things with more than two instances though, or where the documentation would be intended to be reused in other implementations, etc.